### ISSUE TITLE
[HttpFoundation] Add support for SameSite=None cookies

### ISSUE THREAD

**Issue by javiereguiluz** [symfony/symfony#31467]
**Description**  
Google has announced some changes related to users privacy that affect to SameSite cookies introduced in #19104.

```
Starting from Chrome 76 [...] Cookies without the SameSite attribute will 
not be available in a third-party context. Developers will need to declare 
cookies that need to be available on third-party sites to Chrome with 
SameSite=None.

Google says this will allow Chrome users to clear cross-site cookies and 
leave single domain cookies that are used for logins and settings intact.
```

Full details: https://web.dev/samesite-cookies-explained/

**Comment by markitosgv** [comment]
Thanks @javiereguiluz! 

What do you think about using Lax as default cookie value?

**Comment by nicolas-grekas** [comment]
I would consider this a bug fix, should be fixed on 3.4 (same as we consider support for eg new PHP versions bug fixes)

@markitosgv we cannot change the default value without breaking BC.
